courses/views.py

```diff
@@ -154,3 +155,10 @@ def student_can_add_course(request, course_id, student_id):
         student.save()
         messages.success(request, 'You are enrolled in %s' % (course))
     return redirect('/')
+
+
+class CourseEdit(UpdateView):
+    model = Course
+    template_name = "course_edit.html"
+    # context_object_name = "course"
```

Comment:
this is used in the template ... you have a variable named course ... it holds the object that will be edited